typescript/package.json

```diff
@@ -17,20 +19,11 @@
   ],
   "scripts": {
     "build": "echo noop",
-    "build-wasm": "wireit",
-    "watch": "npm run build-wasm --watch",
+    "build-wasm": "wasm-pack build --target web",
+    "watch-wasm": "wasm-pack build --dev --mode no-install --target web",
```

Comment:
After we do an `npm i`, we should have the bits we need so this will allow for fast rebuilds when we are running `npm run playground`.